.github/workflows/e2e-fixture-test.yml

```diff
@@ -0,0 +1,31 @@
+name: E2E Fixture Test
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
+
```

Comment:
We need `permissions` set on the main level, and/or in the `jobs`.

Reply:
Thanks for the feedback! It's now implemented.